### DISCUSSION TITLE
Welcome to ARI4Java Discussions!

### DISCUSSION THREAD

**Discussion by grahambrown11** [l3nz/ari4java#167] [General]
## 👋 Welcome!

Lets try this new feature from GH - Discussions as a place to connect and ask questions :smile:

You're welcome to create new discussions or comment on existing ones. There are 2 categories:

- **_General_** - for anything that's not a question requiring an answer
- **_Q&A_** - for questions

Please start a new discussion if not related to the ones already created.

**Comment by l3nz** [comment]
Hi all!
